vars/kolaTestIso.groovy

```diff
@@ -52,6 +53,20 @@ def call(params = [:]) {
                 archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso-metal4k.tar.xz'
             }
         }
+        if (!params['skipMultipath4k']) {
+            testIsoRuns2["multipath4k"] = {
+                try {
+                    if (params['scenarios4k']) {
+                        shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k --qemu-multipath ${extraArgs4k} --scenarios ${scenarios4k} --output-dir tmp/kola-testiso-metal4k")
+                    } else {
+                        shwrap("cd ${cosaDir} &&  kola testiso -S --qemu-native-4k --qemu-multipath ${extraArgs4k} --output-dir tmp/kola-testiso-metal4k")
+                    }
+                } finally {
+                    shwrap("cd ${cosaDir} && tar -cf - tmp/kola-testiso-multipath4k/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-multipath4k.tar.xz")
+                    archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso-multipath4k.tar.xz'
+                }
+            }
+        }
```

Comment:
ummm, I added it out mostly because the tests I saw the multipath test was done only in 4k and not in the other tests, but I'm not so sure if it  makes sense to test it only in 4k

Reply:
OK gotcha. There's a combinatorial matrix issue here with UEFI, multipath, and metal4k (though that one implies UEFI). I think let's just drop that commit for now and we can revisit. When we move the FCOS pipeline to use `kolaTestIso()`, we'll still get multipath coverage but on metal, which is a more common scenario than metal4k anyway.

Reply:
a ok. Dropping it